test/benchmarks/chains.sol

```diff
@@ -0,0 +1,86 @@
+contract StdCheatsSafe {
```

Comment:
I'm assuming this was copied from somewhere. Can you add a comment with a link to the original source?

Reply:
Gonna need either @chriseth (or maybe @leonardoalt) to provide the source for this.